README.md

```diff
@@ -1 +1,32 @@
-# Restaurant
\ No newline at end of file
+# Restaurant
+
+Restaurant is a website for my coffee shop that has three parts to it. One, a warm and welcoming home page with an image of the restaurant and some good message. And, a menu page showing what we serve at the spot. It also has a contacts page where we've given details of our location and contacts for customers to feel free to reach us.
+
+## Built with
+- Js
+- HTML
+- CSS & Bootstrap
+
+## To get started
+
+- Clone the repository and cd into the library project.
+- ` $ git clone https://github.com/Leon-Mbegera/Restaurant.git` 
+- ` $ cd Restaurant`
```

Comment:
[OPTIONAL]

- [ ] You have a beautiful project, consider adding a screenshot to your README.